ml-conversational-analytic-tool/test_commentAnalysis.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+from commentAnalysis import CommentAnalyzer
+
+
+class TestCommentAnalysis(unittest.TestCase):
+
+    
+    def test_CommentAnalyzer_init(self):
+        analyzer = CommentAnalyzer(['test'])
+        """ positive test """
+        self.assertEqual(analyzer.word_count, {'test': 0})
+        """ negative test """
+        self.assertNotEqual(analyzer.word_count, {'test': 1})
+
+    def test_analyze_comments(self):
+        analyzer = CommentAnalyzer(['test'])
+        """ positive test """
+        result = analyzer.analyzeComment("This is a test comment")
+        self.assertEqual(result, {'test': 1, 'Sentiment': 0.0, 'Code Blocks': 0})
+        """ negative test """
+        self.assertNotEqual(result, {'test': 0, 'Sentiment': 0.0, 'Code Blocks': 0})
+
+    def test_preProcess(self):
+        analyzer = CommentAnalyzer(['test'])
+        result = analyzer.preProcess("This is a test comment")
+        """ positive test """
```

Comment:
Stating positive or negative isn't necessary. It's also not that informative.  Instead I suggest each function have a docstring explaining the test.

Reply:
makes sense!! will change this accordingly